client/landing/gutenboarding/hooks/use-steps.ts

```diff
@@ -37,6 +37,7 @@ export default function useSteps(): Array< StepType > {
 		steps = [
 			Step.IntentGathering,
 			Step.Domains,
+			Step.Emails,
```

Comment:
I don't think we should change this file -- I don't believe this code is used by our current onboarding flow. (Feel free to correct me, but this doesn't feel like it's needed.)